warehouse/query-core/src/main/java/datawave/query/transformer/EventQueryTransformer.java

```diff
@@ -148,4 +148,5 @@ public EventBase transform(Entry<Key,Value> entry) {
         
         return event;
     }
+    
```

Comment:
??

Reply:
Same as above

Reply:
Please revert this class to what it was before.